### PR TITLE
adding documentation

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
+## Development
+
+### Setup
+
+**Requirements**: Node, npm, git.
+
+Run the following terminal commands to get started:
+
+- `git clone https://github.com/mozilla/womenandweb.git`
+- `cd womenandweb`
```
Comment (gideonthomas) [inline README.md:14]: on windows "copy"

**Comment by gideonthomas** [comment]
one nit

File: README.md (hunk 1 above)
Comment (gideonthomas) [inline README.md:14]: that was tired incoherent me talking! ignore :P

Comment (gvn) [inline README.md:14]: LOL

**Pull request merged**